api/caseshare/real-api.ts

```diff
@@ -14,28 +14,25 @@ const logger: JUILogger = log4jui.getLogger('real-api')
 const prdUrl: string = getConfigValue(SERVICES_PRD_API_URL)
 const ccdUrl: string = getConfigValue(SERVICES_CCD_CASE_ASSIGNMENT_API_PATH)
 
-export async function getUsers(req: EnhancedRequest, res: Response) {
+export async function getUsers(req: EnhancedRequest, res: Response, next: NextFunction) {
   try {
     const path = `${prdUrl}/refdata/external/v1/organisations/users?returnRoles=true&status=active`
-    const {status, data}: {status: number, data: any} = await handleGet(path, req)
+    const {status, data}: {status: number, data: any} = await handleGet(path, req, next)
     const permissions = CASE_SHARE_PERMISSIONS.split(',')
     const users = [...data.users]
                   .filter(user => user.roles.some(role => permissions.includes(role)))
                   .map(user => prdToUserDetails(user))
     return res.status(status).send(users)
   } catch (error) {
-    return res.status(error.status).send({
-      errorMessage: error.data,
-      errorStatusText: error.statusText,
-    })
+      next(error)
```

Comment:
The only issue now here is the returned format is no longer the same as expected?

Reply:
Agreed with you on slack its not the case. @paddybasi